template/src/components/LocalhostWarning.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Paragraph, TextLink, Note } from '@contentful/forma-36-react-components';
+
+const LocalhostWarning = () => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        marginTop: '40px'
+      }}>
+      <Note title="It looks like you're running your app locally." style={{ maxWidth: '800px' }}>
+        <Paragraph>
+          If this is your first time running your app, please run `npx
+          @contentful/create-contentful-app create-defintion` in your terminal then{' '}
```

Comment:
While I think this is a good approach, I still feel a bit hesitant about this appDefinition flow. I think for first-timers, this is still pretty confusing and the appDefinition might make more sense when they see the UI version of it.

That being said, we can always gather more feedback once this version is released to see if people are still confused by it.

Reply:
☝️ I agree with this. I'd favor to give two options here:

- a link to documentation explaining the concept and the steps in the ui
- a "alright, you know what ya doing" way promoting the cli command